package.json

```diff
@@ -52,7 +52,7 @@
     "ember-load-initializers": "^2.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-resolver": "^4.0.0",
-    "ember-source": "~3.0.0",
+    "ember-source": "~2.16.0",
```

Comment:
why the ember downgrade?

Reply:
Yes... Why the ember downgrade? No idea, how that one slipped in. Thanks 😂

Reply:
Ah, it was because of ember-try.